tests/system/node/0/config.yaml

```diff
@@ -15,6 +15,10 @@ node:
   # Path to the data directory (if the path doesn't exist it will be created)
   data_dir: .cache
 
+  commons_budget_address: GCOMMONBGUXXP4RFCYGEF74JDJVPUW2GUENGTKKJECDNO6AGO32CUWGU
+  max_size_payload: 1024
```

Comment:
If you set the default value in `Config.d`, then you don't have to manually specify the values here.